### ISSUE TITLE
Restructure how our macros are organized

### ISSUE THREAD

**Issue by eosullivan** [WCSim/WCSim#81]
We currently have two default macros in WCSim: novis.mac and vis.mac. These macros contain all possible detector configuration options, as well as any options which can be controlled with the detector messenger. We should think about restructuring to have a simple default macro file, which can then include other macro files to perform more complicated functions. This would mean having a larger catalogue of macros (for example, having a new macro for every new detector configuration) and either having a large macro with all the possible commands (but which is not the default macro) or utilizing the wiki to explain all the available commands. This could be a potential fix for issue #79 and is related to issue #80.

**Comment by tdealtry** [comment]
#142 looks good. Can I suggest in a future PR having a "macros" (or similar name) directory where all the macros (probably other than WCSim.mac) can live. There are already a lot of macros (and presumably lots more will come into existence) - having a specialist directory will keep the root WCSim directory much clearer & cleaner.

**Comment by eosullivan** [comment]
Yes, agreed. In the future, a separate directory for the macros is probably the way to go.
